src/cpu/aarch64/injectors/jit_uni_eltwise_injector.cpp

```diff
@@ -0,0 +1,2253 @@
+            break;
+        case EQ_UQ:
+            h->fcmeq(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case NGE_US:
+            h->fcmlt(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case NGT_US:
+            h->fcmle(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case NEQ_OQ:
+            h->fcmne(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case GE_OS:
+            h->fcmge(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case GT_OS:
+            h->fcmgt(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case EQ_OS:
+            h->fcmeq(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case LT_OQ:
+            h->fcmlt(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case LE_OQ:
+            h->fcmle(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case NEQ_US:
+            h->fcmne(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case NLT_UQ:
+            h->fcmge(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case NLE_UQ:
+            h->fcmgt(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case EQ_US:
+            h->fcmeq(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case NGE_UQ:
+            h->fcmlt(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case NGT_UQ:
+            h->fcmle(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case NEQ_OS:
+            h->fcmne(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case GE_OQ:
+            h->fcmge(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+        case GT_OQ:
+            h->fcmgt(
+                    PRegS(IDX(p_mask)), p_tmp0 / T_z, vmm_src, compare_operand);
+            break;
+
+        case UNORD_Q:
+        case ORD_Q:
+        case FALSE_OQ:
+        case TRUE_UQ:
+        case UNORD_S:
+        case ORD_S:
+        case FALSE_OS:
+        case TRUE_US:
+        default: assert(!"Unsupported compare mode"); break;
+    }
+}
+
+// Uses injector masks objects: k_mask (>= avx512_common) or vmm_mask (<= avx2).
```

Comment:
This comment seems irrelevant as you always use `sel` and not masked execution.

Reply:
Thank you for the comment. I'll modify comments to reflect the situation on aarch64.
(AArch64 SEL corresponds to x64 VBLENDVPS.)